samples/apps/autogen-studio/autogenstudio/datamodel.py

```diff
@@ -16,7 +16,18 @@
     Enum as SqlEnum,
 )
 
-SQLModel.model_config["protected_namespaces"] = ()
+if hasattr(SQLModel, "model_config"):
+    SQLModel.model_config["protected_namespaces"] = ()
+elif hasattr(SQLModel, "Config"):
```

Comment:
Looks good.
Thanks @Hk669 
Can you confirm you have this tested in 3.11 and sqlmodel@latest ?
After that, should be good to merge in!

Reply:
Sure, let me try this on a windows machine, and confirm you.

Reply:
<img width="789" alt="image" src="https://github.com/user-attachments/assets/24ccf59a-2082-4783-b452-6110e60640d2">

@victordibia its working on windows, `python==3.11` and `sqlmodel==0.0.22`, here is the working status of the application